content/en/blog/_posts/2025-03-01-Container-Stop-Signals.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+title: "Updates to Container Lifecycle in Kubernetes v1.33"
+date: 2025-03-01
+slug: text-for-URL-link-here-no-spaces
```

Comment:
Should the feature owner decide the slug?

Reply:
Yes, I think this is your call to decide what URL would make most sense for this blog 👍